### PR TITLE
Chore (Migration): Set protocol of Hop LP tokens

### PR DESCRIPTION
**Pull request by OjusWiZard**
## Checklist

- [x] Add data migration 15 to set protocol value as 'hop' for the Hop LP tokens

### CODE REVIEW

File: rotkehlchen/data_migrations/migrations/migrations_15.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            'LEFT JOIN history_events as HE on HE.identifier=EE.identifier '
+            'WHERE counterparty=? AND type=? AND subtype=?;', (
+                CPT_HOP,
+                HistoryEventType.RECEIVE.serialize(),
+                HistoryEventSubType.RECEIVE_WRAPPED.serialize(),
+            ),
+        ).fetchall()
+
+    progress_handler.set_total_steps(len(hop_lp_assets))
+
```
Comment (yabirgb) [inline rotkehlchen/data_migrations/migrations/migrations_15.py:39]: the updates will happen fast so I wouldn't worry about setting the steps. Set just one step (update the protocol field in the assets).

Once that is done don't call in fetchall. Iterate over the cursor. This avoids iterating twice the result of the query

File: rotkehlchen/data_migrations/migrations/migrations_15.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        except (WrongAssetType, UnknownAsset) as e:
+            log.error(f'{e!s} when deserializing an EvmToken in data migration 15')
+            progress_handler.new_step(identifier)
+            continue
+
+        progress_handler.new_step(asset.name)
+        GlobalDBHandler.set_token_protocol_if_missing(
+            token=asset,
+            new_protocol=CPT_HOP,
+        )
```
Comment (yabirgb) [inline rotkehlchen/data_migrations/migrations/migrations_15.py:53]: doesn't it fit in one line?

File: rotkehlchen/tests/data_migrations/test_migrations.py [hunk 3]
```diff
@@ -492,6 +505,52 @@ def test_migration_14(
     )
 
 
+@pytest.mark.parametrize('data_migration_version', [14])
+@pytest.mark.parametrize('perform_upgrades_at_unlock', [True])
+def test_migration_15(rotkehlchen_api_server) -> None:
+    """
+    Test migration 15
```
Comment (yabirgb) [inline rotkehlchen/tests/data_migrations/test_migrations.py:512]:
```suggestion
    """Test migration 15
```

**Review by LefterisJP — APPROVED** [review]
LGTM

**Pull request merged**